client/app/components/BoolFilters.js

```diff
@@ -0,0 +1,39 @@
+import React, { Component } from 'react';
+import { View, Image, StyleSheet } from 'react-native';
+import styles from '../config/styles';
+
+export default class BoolFilters extends Component {
+	constructor(props) {
+        super(props);
+    }
+	render() {
+        return (
+            <View style={{ ...styles.container.flexRow }}>
+            	{this.props.veg && 
+	            	<View style={{ ...filterStyles.boolFilters }}>
+				        <Image style={{ width: 30, height: 37 }} source={require("../../assets/images/boolfilters/veg.png")} />
```

Comment:
As with the crests, let's make this dynamically resizable. Use the following props for `Image`:
`resizeMode='contain', sytle={{flex: 1, width: undefined, height: undefined}}`

Reply:
Awesome - will just update that :)